requirements/base.txt

```diff
@@ -26,7 +26,7 @@ sendgrid
 slackclient
 eth-tester
 websockets
-web3==4.0.0b10
+web3==4.4.1
```

Comment:
might be worth a regression test of functionality elsewhere if were bumping 4 minor versions